src/main/kotlin/no/nav/familie/ef/sak/infotrygd/PeriodeService.kt

```diff
@@ -63,7 +63,9 @@ private fun AndelTilkjentYtelse.tilInternPeriode(): InternPeriode = InternPeriod
         personIdent = this.personIdent,
         inntektsreduksjon = this.inntektsreduksjon,
         samordningsfradrag = this.samordningsfradrag,
-        beløp = this.beløp,
+        utgifterBarnetilsyn = 0, // this.utgifterBarnetilsyn TODO
```

Comment:
Har opprettet egen oppgave for dette: https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-8108